src/model/annotation-list.v1.yaml

```diff
@@ -0,0 +1,50 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Annotation list
+type: object
+properties:
+    total:
+        title: Total number of Annotations
+        type: integer
+        minimum: 0
+    items:
+        title: Annotations
+        type: array
+        items:
+            type: object
+            properties:
+                uri:
+                    description: Annotation URI
+                    type: string
+                    format: uri
+                title:
+                    description: Title of the annotated document
```

Comment:
As in the article title? We might want its type+ID too? Maybe we should include the snippet instead.

Reply:
It would be the title of whichever document that has been annotated: Article, blog article, labs post.

If we were to serve up the snippet of the content then this would require us to derive that from the target url which shouldn't be too much trouble but would require us to do a request for each annotation to the relevant service or to store the snippets for all content as part of the annotations service.

Having reviewed the designs for the annotations to be displayed on the profiles page, adding the snippets into the response would not add much value at this stage other than ensuring that the correct title of the annotated document was displayed. As things stand it would be the title of the document at the time that the annotation was recorded.

Reply:
> As things stand it would be the title of the document at the time that the annotation was recorded.

Interesting, so different to the snippet then. We can ignore that for now, and add later if we need it.